### PR TITLE
Admin order index displays all orders

### PR DESCRIPTION
**Pull request by scottalexandra**

### CODE REVIEW

File: spec/feature/authenticated_admin_spec.rb [hunk 1]
```diff
@@ -321,4 +330,20 @@
...
+    item = Item.create(title: "Bacon",
+                       description: "desc",
+                       price: 1000,
+                       status: "hidden")
+    item2 = Item.create(title: "Eggs",
+                        description: "another",
+                        price: 2000)
+    order.line_items.create(item_id: item.id, quantity: 1)
+    order.line_items.create(item_id: item2.id, quantity: 2)
+    order2 = Order.create(user_id: 1)
```
Comment (scottalexandra) [inline spec/feature/authenticated_admin_spec.rb]: Its not useless!

File: app/models/order.rb [hunk 2]
```diff
@@ -20,4 +20,8 @@ def total(line_items)
   def format_order_number(order_id)
     order_id.to_s.rjust(5, "0")
   end
+
+  def self.select_orders_by_status(status, orders)
+    orders.select(status: status)
+  end
```
Comment (rshea303) [inline app/models/order.rb]: method not necessary

File: app/views/admin/orders/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<h1>All Orders</h1>
+
+  Ordered: <%= @orders.where(status: "ordered").count %><br>
+     Paid: <%= @orders.where(status: "paid").count %><br>
+Completed: <%= @orders.where(status: "completed").count %><br>
+Cancelled: <%= @orders.where(status: "cancelled").count %><br>
```
Comment (rshea303) [inline app/views/admin/orders/index.html.erb:6]: would the status be better off within an unordered list than using breaks?

**Comment by rshea303** [comment]
Reviewed updates.

**Pull request merged**